specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateEndpointConnection.json

```diff
@@ -0,0 +1,285 @@
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateEndpointConnectionNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. The private endpoint connection update will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Approve or reject a private endpoint connection with a given name.": {
+            "$ref": "./examples/CosmosDBPrivateEndpointConnectionUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "PrivateEndpointConnections"
+        ],
+        "description": "Deletes a private endpoint connection with a given name.",
+        "operationId": "PrivateEndpointConnections_Delete",
+        "parameters": [
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateEndpointConnectionNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. The private endpoint connection delete will complete asynchronously."
+          },
+          "204": {
+            "description": "Private endpoint connection does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Deletes a private endpoint connection with a given name.": {
+            "$ref": "./examples/CosmosDBPrivateEndpointConnectionDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PrivateEndpointConnectionListResult": {
```

Comment:
Since a number of teams are using these shared definitions, and the schema has been set by NRP (I think?), could we move them somewhere common for other teams to reference directly?

e.g. somewhere in /specification/common-types/?

Reply:
I moved them to common-types